src/server/models/mediaquery.js

```diff
@@ -98,6 +104,16 @@ module.exports.getMediaForTerm = async (dbCollection, academicTerm) => {
   return termMedia;
 };
 
+module.exports.getSubjectAreasForTerm = async (dbCollection, term) => {
+  const query = term !== '' ? { term } : {};
+  const subjectAreasArray = await client
+    .db(DB_DATABASE)
+    .collection(dbCollection)
+    .distinct('subjectArea', query);
```

Comment:
@chern What order does this return the subject areas? Does it return it sorted alphabetical or just the order it was inserted? Can it be alphabetical?

Reply:
@rlorenzo The behavior I've seen has been that distinct() always returns it sorted alphabetical (not insertion order), although I haven't seen any documentation that officially states this